README.md

```diff
@@ -41,4 +41,9 @@ Run `adocPublishToWiki` gradle task
 | wiki.space-key                           | Confluence space key.                                     |               | +                   |
 | wiki.page-id                             | Confluence page id.                                       |               | +                   |
 
+### Excluded Components Configuration
+
+In the `Component-Registry` project, there must be a file named `excludedComponents.txt`. This file should contain a list of components for which the validation of the presence of GAV, DEB, RPM, or Docker will be disabled.
```

Comment:
I don't think it's mandatory. If it's not present then no exceptions are applied